### DISCUSSION TITLE
Alpha 3 🎉

### DISCUSSION THREAD

**Discussion by prayag17** [prayag17/Blink#461] [Announcements]
## What's Changed
- refactor: prepare for alpha 3
- refactor: cleanup
- refactor: remove unused imports
- fix: series page not fetching correct season
- feat: add key binding for macos users #435
- refactor: remove jellyPlayer naming from code #460
- feat: add playlist playback support
- refactor: improve server add process
- feat: show current time while scrubbing video progress #339
- style: increase the sharpness of card images
- feat: add skip recap button
- refactor: cleanup video player
- feat: add chapter navigation buttons to video player #457
- feat: allow changing audio track from video player
- feat: allow changing chapters from a chapter list #457
- refactor: cleanup search route
- refactor: cleanup some components
- refactor: unmount card image if not inView
- refactor: cleanup person route
- style: replace noto-sans with plus-jakarta-sans
- refactor: cleanup login routes
- fix: library items not of same size #380
- refactor: cleanup item.tsx
- fix: series page breaking if season is not loaded
- fix: do not display missing items in series
- refactor: move more files to ts
- refactor: move more files to ts
- refactor: move more files to ts
- refactor: move more files to typescript
- refactor: move more components to ts
- refactore: cleanup __root.tsx
- chore(deps): update dependencies
- fix: collections page not rendering #447
- fix: quick connect not working #445
- fix: playback error if subtitle is not set
- fix: index not being assigned proper value #420
- fix: synced lyrics not working
- Merge pull request #446 from prayag17/renovate/node-22.x-lockfile
- refactor: use saved lyrics
- refactor: change GenreView to genreFilter
- fix(deps): update dependency @types/node to v22.10.1
- Merge pull request #393 from prayag17/renovate/tanstack-router-monorepo
- Merge pull request #424 from prayag17/renovate/tanstack-query-monorepo
- fix(deps): update tanstack-router monorepo
- Merge pull request #426 from prayag17/renovate/node-22.x-lockfile
- fix(deps): update dependency @types/node to v22.10.0
- Merge pull request #437 from prayag17/dependabot/cargo/src-tauri/rustls-0.23.18
- fix(deps): update tanstack-query monorepo to v5.61.4
- Merge pull request #438 from prayag17/renovate/axios-1.x-lockfile
- Merge pull request #439 from prayag17/renovate/vitejs-plugin-react-4.x-lockfile
- chore(deps): update dependency @vitejs/plugin-react to v4.3.4
- refactore: move more components to typescript
- fix: episode and series not playing if episode is not present in nextUp and continueWatching
- feat: show current episode in PlayButton
- fix: episode indexing for absolute numbering indexNumbers in PlayButton
- fix: unable to read Id of undefined in PlayButton
- fix(deps): update dependency axios to v1.7.8
- chore(deps): bump rustls from 0.23.16 to 0.23.18 in /src-tauri
- fix(ci): not uploading debug builds
- refactor(ci): build debug builds in CI
- Merge pull request #425 from prayag17/renovate/serde_json-1.x-lockfile
- Merge pull request #436 from sambartik/macos-universal-dmg
- chore(deps): update dependencies
- refactor: convert more files to typescript
- fix: improper handling of undefined api leading to crashes
- refactor: move all buttons to tsx #338
- Add universal dmg for macOS in release job
- fix: audio queue not moving after 2 changes #182
- fix: playing track from album not updating the queue #182
- refactor: use MediaSegments api for intro/outro skipping
- fix: trickplay images not showing up
- refactor: improve login page UI #338 #202
- chore(release): update latest.json
- fix(deps): update rust crate serde_json to v1.0.133

## New Contributors
* @sambartik made their first contribution in https://github.com/prayag17/Blink/pull/436

**Full Changelog**: https://github.com/prayag17/Blink/compare/v0.0.7-dev...v1.0.0-alpha03

<hr /><em>This discussion was created from the release <a href='https://github.com/prayag17/Blink/releases/tag/v1.0.0-alpha03'>Alpha 3 🎉</a>.</em>

**Comment by Nsbx** [comment]
Just a small comment to say : Thanks.
Even in Alpha this app is way better than "Jellyfin Media Player".
I have two screens, and for strange reason "JMP" freeze and crash so much time ...
And for others reason, the web player doesn't want to load some files.
After 1 hour of watching, no crash.
This is perfect replacement ❤

**Reply by prayag17** [reply]
Hey @Nsbx, thanks for the compliment 😊.

**Comment by Mavarice** [comment]
Any chance an AppImage (or Flatpak) version will be released for those of us on Linux not using a .deb compatible distro?

**Reply by prayag17** [reply]
Tauri's GitHub action image had some issue when Alpha 3 release was published causing it not to build an appimage, but it will be included in next builds.
